resources/sync_gateway_configs/sync_gateway_bucketshadow_low_revs_cc.json

```diff
@@ -14,14 +14,17 @@
             {{ no_conflicts }}
             {{ sg_use_views }}
             {{ num_index_replicas }}
+            {{ username }}
+            {{ password }}
+            {{ cacertpath }}
+            {{ certpath }}
+            {{ keypath }}
             "server":"{{ server_scheme }}://{{ couchbase_server_primary_node }}:{{ server_port }}",
             "bucket":"data-bucket",
-            "username":"data-bucket",
-            "password": "password",
-    	    "revs_limit":20,
-    	    "shadow": {
-		        "server": "{{ server_scheme }}://{{ couchbase_server_primary_node }}:{{ server_port }}",
-		        "bucket": "source-bucket",
+            "revs_limit":20,
+            "shadow": {
+            "server": "{{ server_scheme }}://{{ couchbase_server_primary_node }}:{{ server_port }}",
+            "bucket": "source-bucket",
```

Comment:
bucket name changed, hope tests are not effected with this change. is bucket name need to changed? Can't we use old name?

Reply:
No, it didn't change. It's a part of shadow object which was there earlier too.